everviz/plugins/_crossplot_plugin/crossplot_control_group_plugin.py

```diff
@@ -0,0 +1,161 @@
+from uuid import uuid4
+import dash_html_components as html
+import numpy as np
+from dash.dependencies import Input, Output
+import plotly.graph_objs as go
+from webviz_config import WebvizPluginABC
+
+from everviz.plugins._crossplot_plugin._dash_components._crossplot_callback import (
+    _crossplot_callback,
+)
+from everviz.plugins._crossplot_plugin._dash_components._crossplot_layout import (
+    _get_layout,
+)
+from everviz.plugins._crossplot_plugin._get_indexed_control import (
+    _identify_indexed_controls,
+)
+from everviz.data.load_csv._get_data import get_data
+import dash_core_components as dcc
+
+
+class CrossplotControlGroupPlugin(WebvizPluginABC):
+    def __init__(self, app, data_path, title="Crossplot"):
+        super().__init__()
+        self.title = title
+        self.graph_id = f"graph-{uuid4()}"
+        self.dropdown_x_id = f"dropdown-{uuid4()}"
+        self.dropdown_y_id = f"dropdown-{uuid4()}"
+        self.radio_items_x_id = f"radio-{uuid4()}"
+        self.radio_items_y_id = f"radio-{uuid4()}"
+        self.dropdown_realization_id = f"dropdown-{uuid4()}"
+        self.dropdown_realization_2_id = f"dropdown-{uuid4()}"
+        self.interpolation_id = f"interpolation-{uuid4()}"
+
+        self.data_path = data_path
+        self.set_callbacks(app)
+
+    def add_webvizstore(self):
```

Comment:
What does this function do?  I might not have updated my webviz-config package but it seems this is not overloading anything in the parent class and that it is not used anywhere in the code.

Reply:
It is explained a bit [here](https://github.com/equinor/webviz-config/blob/master/CONTRIBUTING.md#deattaching-data-from-its-original-source), and useful in e.g. scenarios where the data source is local on disk, and not (yet) properly available through an available API.

Based on your question @DanSava, `webviz-config` could perhaps later consider to  deprecate `add_webvizstore` and instead provide an equivalent function decorator making it more transparent to code reviewers/readers that it is a method used by `webviz-config`.